weldx/asdf/tags/weldx/core/dataset.py

```diff
@@ -54,11 +51,6 @@ def to_tree(cls, node: Dataset, ctx):
             "variables": variables,
         }
 
-        # variables = []
-        # for variable_name in node:
-        #    variables.append(netcdf.NetCDFVariable(node[variable_name]))
```

Comment:
Actually, I also thought about using a binary representation. But I guess you favored the other approach to see some metadata in the asdf header, right?